### DISCUSSION TITLE
Module '"@mantine/core"' has no exported member 'Pill'.

### DISCUSSION THREAD

**Discussion by PMayur1504** [mantinedev/mantine#5551] [Q&A]
import React from 'react';
import {Pill} from '@mantine/core';

export type PillProps = {
  children: String;
};

export const PillComp = ({children}: PillProps) => <Pill>{children}</Pill>;

**Comment by rtivital** [comment]
What version of `@mantine/core` do you use?

**Reply by PMayur1504** [reply]
I'm using 7.4.1. But just now I figured out that I'm using mono repository in same project and there I'm using version 6. But now I tried to update that one but it is giving me different error saying version mismatch.

**Reply by rtivital** [reply]
To use v7 components, you must install the latest version.

**Reply by PMayur1504** [reply]
Yes, I need to update version to 7. Thanks for the help.
